src/routes/notLoggedInComponent.svelte

```diff
@@ -0,0 +1,7 @@
+<div class="flex h-[70vh] flex-col items-center justify-center">
+    <div class="border-md rounded-md bg-surface-700 p-4">
+        <h1 class="h1 text-center">
+            Diese Seite steht nur angemeldeten Nutzern zur Verfügung!
```

Comment:
```suggestion
            Melde dich an, um diese Seite zu benutzen!
```
Ich glaube, wir haben solche Nachrichten im Frontend bisher ziemlich Gender-neutral formuliert, dann würde ich das gerne auch so beibehalten.

Könnte man hier auch noch einen Button wie "Zum Login" hinmachen? Wär cool wenns nicht viel Arbeit ist